src/toggle-button/interfaces.ts

```diff
@@ -7,7 +7,7 @@ import { IconProps } from '../icon/interfaces';
 import { BaseComponentProps } from '../internal/base-component';
 import { NonCancelableEventHandler } from '../internal/events';
 
-export interface ToggleButtonProps extends BaseComponentProps, BaseButtonProps {
+export interface ToggleButtonProps extends BaseComponentProps, Omit<BaseButtonProps, 'style'> {
```

Comment:
How about instead of adding and then omitting, move the `style` prop declaration from the `BaseButtonProps` to `ButtonProps`

Reply:
Makes sense. Updating.